portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFolderLocalServiceImpl.java

```diff
@@ -331,6 +334,35 @@ public int getCompanyFoldersCount(long companyId) {
 		return dlFolderPersistence.countByCompanyId(companyId);
 	}
 
+	@Override
+	public long getDLFolderSize(String treePath, long companyId, long groupId) {
```

Comment:
we are on DLFolder file so there is no need to name the kind of folder

public long getFolderSize

see that on this file (DLFolderLocalService)
the names are 

getFile
getFolder
...

it's because we are on the "DL" kind of folder so the "DL" is implicit